packages/patternfly-4/react-table/src/components/Table/ActionsColumn.js

```diff
@@ -38,22 +40,22 @@ class ActionsColumn extends React.Component {
   };
 
   onSelect = (event, onClick) => {
-    const { rowId } = this.props;
+    const { rowData, extra } = this.props;
     event.preventDefault();
-    onClick && onClick(event, rowId);
+    onClick && onClick(event, rowData, extra);
```

Comment:
+1 for passing `rowData`  back. Maybe rename `extra` to `extraParams` for consistency?

Reply:
fixed